src/cpp/r/RExec.cpp

```diff
@@ -425,7 +425,6 @@ Error RFunction::call(SEXP evalNS, bool safely, SEXP* pResultSEXP,
    // verify the function
    if (functionSEXP_ == R_UnboundValue)
    {
-      LOG_ERROR_MESSAGE("Failed to find function: '" + functionName_ + "'");
```

Comment:
Removed as this message is somewhat noisy + the caller should be responsible for logging the error is appropriate anyhow.